src/Umbraco.Infrastructure/Install/FilePermissionHelper.cs

```diff
@@ -53,6 +53,7 @@ public FilePermissionHelper(
         {
             hostingEnvironment.MapPathContentRoot(Constants.SystemDirectories.Bin),
             hostingEnvironment.MapPathContentRoot(Constants.SystemDirectories.Umbraco),
+            hostingEnvironment.MapPathWebRoot(Constants.System.DefaultUmbracoPath),
```

Comment:
Do you recall what this update was for @ronaldbarendse?

Reply:
This was previously using `hostingEnvironment.MapPathWebRoot(_globalSettings.UmbracoPath)`, but it has been removed since, so I'd suggest not re-adding it again:
```suggestion
```